backend/src/graphql/resolver/AdminResolver.test.ts

```diff
@@ -261,6 +263,216 @@ describe('AdminResolver', () => {
+
+          it('all users by "filters === null"', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 4,
+                    userList: expect.arrayContaining(convertObjValuesToArray(allUsers)),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('all users by "filterByActivated === null && filterByDeleted === null"', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                  filters: {
+                    filterByActivated: null,
+                    filterByDeleted: null,
+                  },
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 4,
+                    userList: expect.arrayContaining(convertObjValuesToArray(allUsers)),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('users with unchecked email', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                  filters: {
+                    filterByActivated: false,
+                    filterByDeleted: null,
+                  },
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 1,
+                    userList: expect.arrayContaining([allUsers.garrick]),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('users with deleted account', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                  filters: {
+                    filterByActivated: null,
+                    filterByDeleted: true,
+                  },
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 1,
+                    userList: expect.arrayContaining([allUsers.stephen]),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('no users with deleted account and unchecked email', async () => {
```

Comment:
New describe block.
`filter by deleted account and unchecked email`

```suggestion
          it('finds no users', async () => {
```

Reply:
Cool! 😍
Done.